src/commands/info.js

```diff
@@ -0,0 +1,25 @@
+const { ApifyCommand } = require('../lib/apify_command');
+const { getLocalUserInfo } = require('../lib/utils');
+const chalk = require('chalk');
+
+class InfoCommand extends ApifyCommand {
+    static async run() {
+        const info = await getLocalUserInfo();
+
+        if (info) {
+            const niceInfo = {
+                username: info.username,
+                userId: info.id,
+                token: info.token,
+                proxyPassword: info.proxy.password,
+            };
+            Object.keys(niceInfo).forEach(key => console.log(`${chalk.gray(key)}: ${chalk.bold(niceInfo[key])}`));
+        }
+    }
+}
+
+InfoCommand.description = `
```

Comment:
The first line of description should be one short sentence, otherwise the text is missing e.g. in 'apify help'